readme.md

```diff
@@ -186,82 +237,244 @@ const results = [
+
+#### In `SearchContainer.js`...
 
-> What is the point of all of these `.bind(this)` statements?
+Define an `onSubmitQuery` method in the component definition. It should use `.setState` to set `hasSearched` to `true`.
 
-We are also going to add a `Utils.js` file where we ultimately will make the request to TVMaze.
-For now we'll just hard code the mock data
+```js
+class SearchContainer extends Component {
+  // ...
+  onSubmitQuery(e) {
+    e.preventDefault()
+    this.setState({
+      hasSearched: true
+    })
+  }
+  // ...
+}
+```
+
+> Because the method will be trigger by a submit event, the `e` parameter is automatically populated with an event object.
+
+Because this method will be triggered when the user interacts with the `Search` component, we need to pass `onSubmitQuery` down to `Search` via props. We can do this with methods similarly to how we pass down data values...
+
+```js
+class SearchContainer extends Component {
+  // ...
+  render(){
+    const toRender = this.state.hasSearched
+      ? <Results shows={this.state.shows} />
+      : <Search query={this.state.query} onSubmitQuery={this.onSubmitQuery} />
+    return <div>{toRender}</div>
+  }
+}
+```
+
+We also need to update the `constructor` so that the context of `onSubmitQuery` is preserved (i.e., `this` = `SearchContainer`) when it is later triggered in `Search`...
+
+```js
+class SearchContainer extends Component {
+  constructor(props){
+    super(props)
+    this.state = { ... }
+    this.onSubmitQuery = this.onSubmitQuery.bind(this)
+  }
+  // ...
+}
+```
 
-#### [SearchContainer component](https://github.com/ga-wdi-exercises/react-tvmaze/commit/1c896c5a975ea9d1f6fd07bbd655caf1d1f9f9ae)
+#### In `Search.js`
 
-### Replace Mock Data with an AJAX request
-Finally, we will replace the mock data with an actual AJAX request and update the `onSubmitQuery` to update state in handling the resolved promise
+Now we have to update the `Search` UI so that when a search is submitted, the view is switched from `Search` to `Results`. In this situation that means triggering the previously-defined `onSubmitQuery` method, which we have passed down from `SearchContainer` via props...
+
+```js
+class Search extends Component {
+  render() {
+    return (
+      <form onSubmit={this.props.onSubmitQuery}>
+        <input type="text" placeholder="Enter search term" />
+        <button type="submit">Submit</button>
+      </form>
+    )
+  }
+}
+```
+
+When your done with this section, [your code should look something like this](https://github.com/ga-wdi-exercises/react-tvmaze/tree/07a579736dd40488dbab943c695e329ab61ae2ba).
+
+## Replace Mock Data with an AJAX Request (10 minutes / 2:30)
+
+Finally, we will replace the mock data with an actual AJAX request and update the `onSubmitQuery` to update state in handling the resolved promise.
 
 #### [AJAX request](https://github.com/ga-wdi-exercises/react-tvmaze/commit/8bb4f4edd5a98261b2a89116e40ca20e9f025269)
 
```

Comment:
are they just copying the code in the link? trying to figure it out on their own? or are you showing them this? lil unclear